lib/travis/build/script/smalltalk.rb

```diff
@@ -0,0 +1,26 @@
+module Travis
+  module Build
+    class Script
+      class Smalltalk < Script
+        DEFAULTS = {}
+
+        def setup
+          super
+          sh.cmd 'sudo apt-get install --no-install-recommends libc6:i386 ' +
+                   'libuuid1:i386', retry: true
+          sh.if '-z "$SMALLTALK"' do
+            sh.export 'SMALLTALK', 'Squeak5.0'
+          end
+          sh.cmd "export PROJECT_HOME=\"$(pwd)\""
+          sh.cmd "cd $HOME"
+          sh.cmd "wget -q -O filetreeCI.zip https://github.com/hpi-swa/filetreeCI/archive/master.zip"
+          sh.cmd "unzip -q -o filetreeCI.zip"
+          sh.cmd "cd filetreeCI-*"
```

Comment:
In general, I advise against use of `cd` to move around. If you need to change working directories, try `pushd`/`popd` in pairs so that our commands do not affect the user's working directory.